### ISSUE TITLE
[Release] Ability for Fiscal Hosts to "unhost" Collectives 

### ISSUE THREAD

**Issue by shannondwray** [opencollective/opencollective#6135]
https://github.com/opencollective/opencollective/issues/3330

**Comment by BenJam** [comment]
https://opencollective.com/opencollective/updates/freeze-and-unhost-collectives

**Comment by shannondwray** [comment]
From @BenJam's Release Draft on Open Collective - We've added the ability to freeze and un-host Collectives for host administrators.
 In order to protect hosts from abuse, and to support Collectives wishing (or needing) to migrate to another host we have added the ability for host admins to self-manage their relationships with their Collectives.



Freezing a Collective will block all financial activity within that Collective. Contributions and expenses will be removed from the Collective's public profile, and admins will be informed of why the action was taken. We've used this within our own host networks to stop fraudulent contributions and stop rogue administrators while we resolve situations with the communities impacted.

We've also added the ability to un-host a collective, which can be used to free a Collective to choose another host or to become an [independent collective](https://docs.opencollective.com/help/independent-collectives/about-independent-collectives). Before un-hosting, a Collective's [balance must be zeroed](https://docs.opencollective.com/help/collectives/collective-settings/zero-collective-balance) by paying expenses, making contributions to other Collectives or gifting the balance to the host.

Both of these options are available from your host dashboard under 'Hosted Collectives'.

📘 Read more about freezing a Collective
📙 Read more about un-hosting a Collective
✉️ Contact support
📣 Leave feedback below
